salt/metalk8s/orchestrate/etcd.sls

```diff
@@ -33,7 +33,7 @@ Check etcd cluster health for {{ node }}:
       - minion_id: {{ node }}
     - attemps: 5
```

Comment:
Unrelated to your PR: we should correct our `module.run` to use the keyword `attempts` and not `attemps`.

Reply:
Oh indeed, feel free to create a ticket :stuck_out_tongue: